### PR TITLE
*: increase the performance of loading stats

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:

ref https://github.com/pingcap/docs/pull/17895

- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1002,7 +1002,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值："json,blob,mediumblob,longblob"
+- 默认值："json,blob,mediumblob,longblob,mediumtext,longtext", 在 v8.2 之前的版本，默认值为 "json,blob,mediumblob,longblob"
```
Comment (hfxsd) [inline system-variables.md:1005]:
```suggestion
- 默认值："json,blob,mediumblob,longblob,mediumtext,longtext"。在 v8.2.0 之前的版本，默认值为 "json,blob,mediumblob,longblob"。
```

**Review by songrijie — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -572,8 +572,8 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 ### `stats-load-concurrency` <span class="version-mark">从 v5.4.0 版本开始引入</span>
 
 + TiDB 统计信息同步加载功能可以并发处理的最大列数
-+ 默认值：5
-+ 目前的合法值范围：`[1, 128]`
++ 默认值：0, 在 v8.2 之前，默认值为5
++ 目前的合法值范围：`[0, 128]`, 0 为自动模式，根据机器情况，自动调节并发度, 在 v8.2 之前，最小值为 1
```
Comment (hfxsd) [inline tidb-configuration-file.md:576]:
```suggestion
+ 默认值：0。在 v8.2.0 之前，默认值为 5。
+ 目前的合法值范围：`[0, 128]`。0 为自动模式，根据服务器情况，自动调节并发度。在 v8.2.0 之前，最小值为 1。
```

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -572,8 +572,8 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 ### `stats-load-concurrency` <span class="version-mark">从 v5.4.0 版本开始引入</span>
 
 + TiDB 统计信息同步加载功能可以并发处理的最大列数
-+ 默认值：5
-+ 目前的合法值范围：`[1, 128]`
++ 默认值：0。在 v8.2.0 之前，默认值为 5。
++ 目前的合法值范围：`[0, 128]`。0 为自动模式，根据服务器情况，自动调节并发度。在 v8.2.0 之前，最小值为 1。
```
Comment (lilin90) [inline tidb-configuration-file.md:576]:
```suggestion
+ 默认值：`0`。在 v8.2.0 之前，默认值为 `5`。
+ 目前的合法值范围：`[0, 128]`。`0` 为自动模式，根据服务器情况，自动调节并发度。在 v8.2.0 之前，最小值为 `1`。
```

File: system-variables.md [hunk 4]
```diff
@@ -1002,7 +1002,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
-- 默认值："json,blob,mediumblob,longblob"
+- 默认值："json,blob,mediumblob,longblob,mediumtext,longtext"。在 v8.2.0 之前的版本，默认值为 "json,blob,mediumblob,longblob"。
```
Comment (lilin90) [inline system-variables.md:1005]:
```suggestion
- 默认值："json,blob,mediumblob,longblob,mediumtext,longtext"。在 v8.2.0 之前，默认值为 "json,blob,mediumblob,longblob"。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**